content/en/docs/virtualization.md

```diff
@@ -334,3 +334,121 @@ windows.2k22 | Microsoft Windows Server 2022
 windows.2k22.virtio | Microsoft Windows Server 2022 (virtio)
 windows.2k25 | Microsoft Windows Server 2025
 windows.2k25.virtio | Microsoft Windows Server 2025 (virtio)
+
+
+## Virtual Routers
+
+Since version v0.27.0, Cozystack provides a virtual routers functionality.
+This feature allows you to create a virtual router that can be used to route traffic between different networks.
+Virtual router backed by a virtual machine instance with a special configuration.
+
+
+### Virtual Router Creation
+
+You can create a virtual machine instance using standard `vm-instance` and `virtual-machine` packages.
+Later, you need to contact your administrator to enable the virtual router functionality for this instance.
+
+### Enabling Virtual Router functionality for the Tenant
+
+These actions should be done by Cozystack administrator and are not available for end-users yet.
+
+First, you need to disable anti-spoofiging protection for the vm-instance:
+
+```
+kubectl patch virtualmachines.kubevirt.io virtual-machine-example --type=merge -p '{"spec":{"template":{"metadata":{"annotations":{"ovn.kubernetes.io/port_security": "false"}}}}}'
+```
+
+Then restart the virtual machine:
+
+```
+virtctl stop virtual-machine-example
+virtctl start virtual-machine-example
+```
+
+Get IP-address for your virtual router:
+```
+kubectl get vmi
+```
+
+After that you can configure custom routes for the tenant namespace:
+
+Run
+
+```
+kubectl edit namespace tenant-example
+```
+
+
+and add annotation as following:
+```
+ovn.kubernetes.io/routes: |
+  [{
+    "dst": "10.10.13.0/24",
+    "gw": "10.244.3.12"
+  }]
+```
+
+Custom routes will be applied to all pods in the tenant namespace.
+
+
+## Virtual Routers
```

Comment:
Starting from here, the whole document repeats.

Reply:
resolved